src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -520,9 +520,9 @@ public partial class NameValueHeaderValue : System.ICloneable
     }
     public partial class NameValueWithParametersHeaderValue : System.Net.Http.Headers.NameValueHeaderValue, System.ICloneable
     {
-        protected NameValueWithParametersHeaderValue(System.Net.Http.Headers.NameValueWithParametersHeaderValue source) : base (default(System.Net.Http.Headers.NameValueHeaderValue)) { }
-        public NameValueWithParametersHeaderValue(string name) : base (default(System.Net.Http.Headers.NameValueHeaderValue)) { }
-        public NameValueWithParametersHeaderValue(string name, string value) : base (default(System.Net.Http.Headers.NameValueHeaderValue)) { }
+        protected NameValueWithParametersHeaderValue(System.Net.Http.Headers.NameValueWithParametersHeaderValue source) : base (default(string)) { }
+        public NameValueWithParametersHeaderValue(string name) : base (default(string)) { }
+        public NameValueWithParametersHeaderValue(string name, string value) : base (default(string)) { }
```

Comment:
Why is the ref file changing as part of this change?

Reply:
I had a temporary API change at one point, and the ref generator picked up this change along with it. Kept it as it fixes an inconsistency.

These are [documented APIs](https://docs.microsoft.com/en-us/dotnet/api/system.net.http.headers.namevaluewithparametersheadervalue.-ctor?view=netcore-3.1) but were not in the ref. Was documentation wrong?

Reply:
> These are documented APIs but were not in the ref. Was documentation wrong?

The changes in the ref appear to just be what base method is called, and that base method invocation is purely there to satisfy the compiler; it doesn't actually impact the publicly exposed signature.  Am I misunderstanding?

Reply:
Oof, review only showed the 1 line being removed. You're correct, the only change is to the base ctor.

I don't think it's a big deal but can revert if you would like to avoid this in a HTTP/3 commit.